apps/web/components/Integration/IntegrationData.tsx

```diff
@@ -1000,5 +1000,26 @@ export class AppComponent {
         </Link>
       </>
     ),
+    '2) Add Import Button': ({ accessToken, projectId, templateId }) => (
+      <>
+        <ModifiedText>
+          Now add Import Button provided by <Code>@impler/react</Code> which opens the Widget
+        </ModifiedText>
+        <CodeBlock
+          height={HEIGHTS.WITH_TEXT}
+          code={`
+import { useImpler } from "@impler/react";
+
+const { showWidget, isImplerInitiated } = useImpler({
+  projectId: "${projectId}",
+  templateId: "${templateId}",
+  accessToken: "${accessToken}",
+});
+
+<button disabled={!isImplerInitiated} onClick={showWidget}>Import</button>`}
+          language="javascript"
+        />
+      </>
+    ),
```

Comment:
@thisismayuresh It's react code, please update as it's in Bubble section.

Reply:
Thanks for pointing that out, I will update it.